test/files/run/t10650b/Test_2.scala

```diff
@@ -0,0 +1,13 @@
+object Test extends App {
```

Comment:
I don't see any additional value in also testing this under separate compilation - maybe I'm missing something?

Reply:
You may be right. I simply wrote tests similar to the ones for https://github.com/scala/bug/issues/9437

Reply:
So should I remove this test? If so, then it probably also makes sense to remove `t9437b`.

Reply:
Ah I see. I don't see the value in separate compilation in the existing tests either. When using a named paramter (`a = 1, `_` = "2"` etc) in Scala code on a Scala-defined method, the Scala compiler doesn't look at the classfile attribute holding the parameter names. The information is obtained from the "pickle", the serialized chunk of the symbol table. So I'd say let's remove the additional test in this PR.

Reply:
Yes, let's remove those tests.